src/index.html

```diff
@@ -7,12 +7,62 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=" "
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="card image"
+      />
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">
+        Product code:
+        <span>195434</span>
+      </p>
+      <div class="card__content align">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="review">Reviews: 5</p>
+      </div>
+      <div class="card__content price">
+        <p class="price__label">Price:</p>
+        <p class="price__value">$2,199</p>
+      </div>
+      <a
+        class="card__product-link"
```

Comment:
Add the blank line between the blocks
```suggestion
    >
      <img
        class="card__image"
        src="./images/imac.jpeg"
        alt="card image"
      />
      
      <h2 class="card__title">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
      
      <p class="card__code">
        Product code:
        <span>195434</span>
      </p>
      
      <div class="card__content align">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
        <p class="review">Reviews: 5</p>
      </div>
      
      <div class="card__content price">
        <p class="price__label">Price:</p>
        <p class="price__value">$2,199</p>
      </div>
      
      <a
        class="card__product-link"
```